### ISSUE TITLE
Upload of large file (1 Gb) stalled on last shard...

### ISSUE THREAD

**Issue by lunokhod** [Storj/core#214]
I am trying to upload a large file (~1 Gb) using the storj command line interface. Everything appeared to work fine, with sharding, getting contracts, and then uploading the shards. However, with 122 shards, the client just hung on the last one
~~~~
 [info]   Shard transfer completed! 3 remaining...
 [info]   Shard transfer completed! 2 remaining...
 [info]   Shard transfer completed! 1 remaining...
~~~~
Its been like this for me than a half hour, and there is no indication that anything is going to happen. (Note that I have successfully uploaded smaller files).

Version: 1.3.1

**Comment by super3** [comment]
The next version should fix this. Better error handling, and probably fixing some farmer side issues that cause that last shard not to complete.

**Comment by bookchin** [comment]
Fixed by `1.3.2`
